common/supplemental/supplementalData.xml

```diff
@@ -1647,7 +1647,7 @@ XXX Code for transations where no currency is involved
 		<language type="ha" scripts="Arab Latn"/>
 		<language type="ha" territories="NE NG" alt="secondary"/>
 		<language type="hai" scripts="Latn"/>
-		<language type="hak" scripts="Hans"/>
+		<language type="hak" scripts="Hans Hant" territories="TW"/>
```

Comment:
Also, we could enforce that if a language is ever written in multiple scripts (aside from non-standard use, like Shavian with English), that the population data must have the script, eg nan_Hant in TW and nan_Hans in CN. Enforce that with tests. Then it would be crystal clear.

Reply:
see https://unicode-org.atlassian.net/browse/CLDR-11224 for an item (needs design) on making crystal clear the status of 'multi script' locales.

Reply:
This field is alphabetically ordered in the generating script. It would be the same output if it was ordered by usage. if we presume 100% Hans in CN and 100% Hant in TW, there are more Hans speakers of both Hakka and Min Nan.

Reply:
The question is always written usage, which is hard to determine as we know.

Reply:
Mind if I follow up on designing that in another change? The non-idempotence of the aforementioned scripts is blocking a lot of PRs waiting on my server.